tests/test_misc.py

```diff
@@ -144,3 +145,37 @@ def test_unordered_dict(init_func):
                'confluent.monitoring.interceptor.topic': 'confluent-kafka-testing',
                'confluent.monitoring.interceptor.icdebug': False
                })
+
+
+# global variable for on_delivery call back function
+seen_delivery_cb = False
+
+
+def test_topic_config_update():
+    confs = [{"message.timeout.ms": 600000, "default.topic.config": {"message.timeout.ms": 1000}},
+             {"message.timeout.ms": 1000}, {"default.topic.config": {"message.timeout.ms": 1000}}]
+
+    def on_delivery(err, msg):
+        # Since there is no broker, produced messages should time out.
+        global seen_delivery_cb
+        seen_delivery_cb = True
+        assert err.code() == confluent_kafka.KafkaError._MSG_TIMED_OUT
+
+    for conf in confs[:]:
```

Comment:
what is the `[:]` for?

Reply:
I wanted to be really explicit about the fact its all the configs not just some.

Reply:
It does the opposite, I'm afraid. Slicing is typically used to get a subset of something.